content/channels/options/deltas.textile

```diff
@@ -44,6 +44,38 @@ There is no constraint on how many publishers or subscribers there are. If there
 
 If a delta is generated and it results in a difference that is not appreciably smaller than the original message, or is larger than the original message, for example if successive messages are completely different, then the delta will not be sent. Clients will receive the original, unprocessed message.
 
+blang[javascript,nodejs].
+
+  h2(#vcdiff). Install vcdiff decoder
+
+  A Vcdiff decoder written in pure JavaScript. The vcdiff decoder is a JavaScript library that decodes delta-compressed messages, allowing clients to reconstruct full messages from the small "diffs" sent by Ably to reduce bandwidth usage.
```

Comment:
I think we can make this a lot shorter since the context of what it does is already on the page.

Reply:
See 40c0260b0c2838ccf68c636df0836618f1150c00